lib/modules/apostrophe-modal/public/js/modal.js

```diff
@@ -453,6 +456,16 @@ apos.define('apostrophe-modal', {
       // we need a gap of time to trigger the transition
       setImmediate(function() {
         $wrapper.find('[data-modal-content]').removeClass('apos-modal-slide-current');
+        var fired = false;
+        $content.one('webkitTransitionEnd otransitionend oTransitionEnd msTransitionEnd transitionend', function() {
+          if (fired) {
+            // Has been seen firing twice in nightwatch tests, in spite of `one`
```

Comment:
chrome runs both `webkitTransitionEnd` and `transitionend` events simultaneously